eng/pipelines/templates/steps/apiview-ui-tests.yml

```diff
@@ -53,6 +53,7 @@ steps:
     APIVIEW_AUTHENTICATIONSCHEME: "Test"
     APIVIEW_SENDGRID__KEY: "Test"
     APIVIEW_APIKEY: "$(azuresdk-apiview-apikey)"
+    APIVIEW_CONNECTIONSTRINGS__APPCONFIG: $(apiview-appconfig-connectionstring)
```

Comment:
Not directly relevant here, but could we pair a single keyvault with each apiview instance to load config secrets on startup, that way secrets don't have to be passed in? Or is the env method easier to work with in an asp environment?

Reply:
this is only for test instance we run on pipeline. Production and deployed instances use KV and App configuration